content/src/fieldConfig/cannabis-priority-status-tab2.json

```diff
@@ -1,6 +1,7 @@
 {
   "cannabisPriorityStatus": {
     "noPriorityStatusText": "Based on your answers, you are not eligible for any priority status. You can mark this task as completed.",
+    "greenBoxHeadingText": "",
```

Comment:
[sand] Can we delete this?

Reply:
I added this in case of override. I'll reach out to content to see if they want to define a header other than the default here.